conda_smithy/templates/appveyor.yml.tmpl

```diff
@@ -47,12 +47,15 @@ install:
     {% if build_setup -%}
     {{ build_setup }}{% endif %}
 
+    # testing purposes: get conda-build from defaults
+    - cmd: conda install -n root --quiet --yes -c defaults conda-build
```

Comment:
Remove this line before merging (version constraint should come from conda-forge-build-setup)